### PR TITLE
Fix/add links in READMEs

### PR DESCRIPTION
**Pull request by anoruxylene**
This is just a casual PR fixing some links in READMEs. 

Additionally, I've found the following dead links in Project's Wiki:
- Under [Using Confluent Kafka client](https://github.com/lovoo/goka/wiki/Tips#using-confluent-kafka-client) section the [github.com/lovoo/goka/kafka/confluent](https://github.com/lovoo/goka/tree/master/kafka/confluent) link leads to 404. 
- Under [Add monitoring and statistics](https://github.com/lovoo/goka/wiki/Tips#add-monitoring-and-statistics) section the [example](https://github.com/lovoo/goka/tree/master/examples/monitoring) link leads to 404. It should probably be [example](https://github.com/lovoo/goka/tree/master/examples/8-monitoring)

### CODE REVIEW

**Review by frairon — APPROVED** [review]

**Comment by anoruxylene** [comment]
@frairon Hey, how do you run this thingy? 
> 3 workflows awaiting approval
This workflow requires approval from a maintainer. [Learn more about approving workflows.](https://docs.github.com/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks)

**Pull request merged**
